src/status_im2/contexts/chat/messages/avatar/view.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im2.contexts.chat.messages.avatar.view
+  (:require [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.core :as quo]
+            [status-im2.contexts.chat.messages.avatar.style :as style]))
+
+(defn avatar
+  [from size]
+  (let [display-name (first (rf/sub [:contacts/contact-two-names-by-identity from]))
+        contact      (rf/sub [:contacts/contact-by-address from])
+        photo-path   (when-not (empty? (:images contact)) (rf/sub [:chats/photo-path from]))
+        online?      (rf/sub [:visibility-status-updates/online? from])]
+    [rn/touchable-without-feedback {:on-press #(rf/dispatch [:chat.ui/show-profile from])}
+     [rn/view {:padding-top 2 :width (style/size->width size)}
```

Comment:
can this not be done with some form of flex so that size is based on the user-avatar component which already has sizes managed? 🤔

Reply:
I can try, maybe we don't have to specify a width

Reply:
removed